source/PlayerInfoPanel.cpp

```diff
@@ -100,7 +101,8 @@ namespace {
 
 
 PlayerInfoPanel::PlayerInfoPanel(PlayerInfo &player)
-	: player(player), canEdit(player.GetPlanet())
+	: player(player), canEdit(player.GetPlanet()
+			&& !player.GetPlanet()->Attributes().count("requires: gaslining"))
```

Comment:
What is the purpose of this change? If the player is landed on a planet, then they're landed on a planet and can edit their fleet ordering, always. That behavior is not to be conditional.

If there is an issue where the player can take some ship that is not landed and then make that ship into their flagship and launch from it, the place to handle that is not here.

Reply:
I think reordering ships while "landed" on a gas planet should not be allowed since you cannot really land on a gas planet. If this is allowed, then I see no reason to disallow reordering the fleet mid-flight (except for changing the flagship, of course).